laterpay/application/Controller/Frontend/Post.php

```diff
@@ -496,6 +496,44 @@ public function check_user_access( $has_access, $post_id = null ) {
     }
 
     /**
+     * Prefetch get posts and exclude free posts with premium content from it
+     *
+     * @wp-hook template_redirect
+     *
+     * @return void
+     */
+    public function prefetch_get_posts() {
```

Comment:
strange method, contains two verbs. @AliaksandrVahura-ScienceSoft  please fix name and description